packages/legend-art/style/base/_panel.scss

```diff
@@ -252,6 +256,18 @@
 
   &__section {
     padding: 0 1rem;
+
+    .input-group__error-message {
+      max-width: 45rem;
+    }
+  }
+
+  &__icon {
```

Comment:
only `PanelFormTextEditor` inputs have the qualification that their max-width is 45rem so the dropdown of an invalid error i put the edit in here as well bc otherwise the red space will expand to the entirety of the default 100% width 🤔